### ISSUE TITLE
Performance Indicator Questions

### ISSUE THREAD

**Issue by YiboSun219** [LTTdouble/SRDNet#5]
Hello author, first of all, I acknowledge your work. However, I have a question. I trained your model using the Houston dataset, but the result is only 44.6908dB, which is far lower than the ERCSR algorithm. I'm using an A800 graphics card for training. I suspect it might be because we split the dataset differently. I divided the Houston dataset into training and test sets at an 80% ratio, and then manually transferred 10% of the data from the training set to the validation set. My hyperparameters are the same as the code you provided. How can I solve this problem?
